content/docs/user-guide/programming/memory/allocators.md

```diff
@@ -37,8 +35,8 @@ The following diagram illustrates the hierarchy of AZ memory allocators.
 To apply an allocator to your class, use the `AZ_CLASS_ALLOCATOR` macro in your class or directly call `AZ::AllocatorInstance`<*some\_allocator*>.
 
 AZCore relies on `AZ_CLASS_ALLOCATOR` to specify the default allocator for the class or on explicit `azcreate` and `azdestroy` calls that specify the allocator in their signature.
-+ If your class does not implement `AZ_CLASS_ALLOCATOR` and you call `new` or `delete`, `new` or `delete` calls use the global `operator new` or `operator delete`.
-+ If your class does not implement `AZ_CLASS_ALLOCATOR` and you call `aznew`, you must implement a `new` operator that uses the `aznew` call signature.
++ If your class does not implement `AZ_CLASS_ALLOCATOR` and you call `new` or `delete`, `new` or `delete` calls, the calls will use the global `operator new` or `operator delete`.
++ If your class does not implement `AZ_CLASS_ALLOCATOR` and you call `aznew`, a compile error will trigger indicating that the operator new must be overwritten.
```

Comment:
I'm not sure if you want to highlight just 'new' or 'operator new' here to be consistent with how the previous line highlights those terms.
```suggestion
+ If your class does not implement `AZ_CLASS_ALLOCATOR` and you call `aznew`, a compile error will trigger indicating that the operator `new` must be overwritten.
```

Reply:
Actually I want to highlight the `operator new` function here as that is the name of the function the class must override: https://en.cppreference.com/w/cpp/memory/new/operator_new

Reply:
So the code should be updated to place "operator new" in backticks.
```suggestion
+ If your class does not implement `AZ_CLASS_ALLOCATOR` and you call `aznew`, a compile error will trigger indicating that the `operator new` must be overwritten.
```